website/src/components/index-page.tsx

```diff
@@ -209,6 +210,7 @@ export function IndexPage(): ReactElement {
       <div className={FeatureWrapperClass}>
         <Hero />
       </div>
+      <RunAnywhereSection className="mx-4 my-6 md:mx-6" />
```

Comment:
We can apply this suggestion and merge it to main after the deploy perview isn't needed anymore.

```suggestion
```